igc_lib.py

```diff
@@ -19,6 +19,9 @@
 import re
 from Bio.Alphabet import Alphabet
 from Bio.HMM.MarkovModel import MarkovModelBuilder
+from xml.dom.minidom import parse
```

Comment:
You don't need this line, as you're referring to `parse` via `xml.dom.minidom.parse`